Dockerfile

```diff
@@ -9,6 +9,8 @@ COPY vendor/ vendor/
 # Build
 RUN if [ "$(uname -m)" = "ppc64le" ]; then \
         CGO_ENABLED=0 GOOS=linux GOARCH=ppc64le go build -a -o manager ./cmd/manager; \
+    elif [ "$(uname -m)" = "aarch64" ]; then \
+        CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -a -o manager ./cmd/manager; \
```

Comment:
make sense to just make this an envvar that gets picked up?

Reply:
Yes, it definately makes sense, however referencing [moby/issue#29110](https://github.com/moby/moby/issues/29110#issuecomment-453718228) I don't think docker currently support the function that set ENV as the result of the shell command (in our case `uname -m`). I think this is also the reason why PowerPC guys keep the coding style of branch as`if ["$(uname -m)" = ARCH]` stuff.

Reply:
if we use `RUN export MACHINE_ARCH=$(uname -m) && if [$MACHINE_ARCH]=ppc64le/aarch64/x86_64]; then blablabla`, I think it is a little bit redundant isn't it?